### PR TITLE
Add actions for the meeting and protocol debug views

### LINKED ISSUE #4799: Debugging views SPV als Aktion für Manager sichtbar machen
Die SPV hat im Moment einige Views mit denen Probleme bei `sablon` und `docxompose` einfacher analysiert werden können, siehe https://docs.onegovgever.ch/admin-manual/meeting/debugging/. Diese sind jedoch im Moment "versteckt", sprich die View muss manuell als Request-URL angegeben werden (z.B. in einem Browser).

Diese Debug-Views sollen neu auch als Aktion sichtbar gemacht werden. Aber nur für `Manager` sprich geschützt mit der Permission `Manage portal` Permission. Als Beispiel eine neue Aktion für eine Sablon-Vorlage:
screen shot 2018-08-28 at 16 14 26

Für folgende Views müssen Aktionen auf den erwähnten Inhaltstypen erstellt werden:

**`fill_meeting_template`**
- Verfügbar auf Sablon-Vorlagen (im Vorlagen-Ordner)
- Name Aktion: Beispiels-Sitzungsdaten einfüllen

**`download_protocol_json`**
- Verfügbar auf Sitzungen
- Name Aktion: JSON Sitzungsdaten herunterladen

**`debug_docxcompose`**
- Verfügbar auf Sitzungen
- Name Aktion: Dateien für docxcompose herunterladen

**`debug_excerpt_docxcompose `**
- Verfügbar auf einem Traktandum
- Name Aktion: Dateien für docxcompose herunterladen
- Soll in den Aktionen des Traktandums erscheinen
screen shot 2018-08-28 at 16 30 02

- [ ] Dokumentation aktualisieren
- [ ] alle Aktionen mit `cmf.ManagePortal` schützen (Relevant fürs darstellen der Aktion)
- [ ] prüfen, dass auch alle Views mit der `cmf.ManagePortal` Permission geschützt sind (Relevant fürs ausführen der Aktion)

### PR DESCRIPTION
**Pull request by njohner**
We add 4 actions pointing to the meeting and protocol debugging views:
* `fill_meeting_template`
* `download_protocol_json`
* `debug_docxcompose`
* `debug_excerpt_docxcompose`
The first three are object buttons while the last one is for agendaitems. All actions and views are protected with the `cmf.ManagePortal` permission.

The `debug_excerpt_docxcompose` is placed in the menu of the agendaitem:

screen shot 2018-09-05 at 15 53 00

resolves #4799

### CODE REVIEW

**Comment by njohner** [comment]
It seems we don't test manager menus. Should we add some tests for that?

File: opengever/core/profiles/default/actions.xml [hunk 1]
```diff
@@ -655,6 +655,42 @@
       <property name="visible">True</property>
     </object>
 
+    <object name="fill_meeting_template" meta_type="CMF Action" i18n:domain="opengever.core">
+      <property name="title" i18n:translate="">Fill meeting template</property>
+      <property name="description" i18n:translate="" />
+      <property name="url_expr">string:${object_url}/fill_meeting_template</property>
+      <property name="icon_expr" />
+      <property name="available_expr">python:here.restrictedTraverse('@@plone_interface_info').provides('opengever.meeting.sablontemplate.ISablonTemplate')</property>
```
Comment (deiferni) [inline opengever/core/profiles/default/actions.xml]: I think these actions should not go into the generic `actions.xml` but into the `[type definition].xml` as at the moment they are defined for only one type. You can also define type-specific actions for one type only, see https://github.com/4teamwork/opengever.core/blob/master/opengever/core/profiles/default/types/opengever.dossier.businesscasedossier.xml vor an example.

File: opengever/meeting/browser/meetings/agendaitem.py [hunk 2]
```diff
@@ -554,8 +557,11 @@ def _get_excerpt_doc_by_uuid(self, doc_uuid):
                 return doc
         return None
 
+    def is_debug_allowed(self):
```
Comment (deiferni) [inline opengever/meeting/browser/meetings/agendaitem.py]: This is a detail, but i would suggest to rename this method to `is_manager` as that is closer to the vocabulary we use for that user. Also the method checks for the `ManagePortal` permission, and not for e.g. a `Debug` permission or debug flag. What do you think?

File: docs/public/admin-manual/meeting/debugging.rst [hunk 3]
```diff
@@ -1,27 +1,31 @@
...
-  analysieren.
+- Inhaltstyp Sitzung: Aktion ``Dateien für docxcompose herunterladen``
+  ermöglicht es die ``docx`` Dateien, die zum Protokoll zusammengesetzt
+  werden, separat in einem ``zip`` herunterzuladen.
+  Dies ermöglicht es Fehler im docxcompose einfacher zu analysieren.
+  Die entsprechende Plone-View ist ``debug_docxcompose``.
 
-- Inhaltstyp Traktandum: ``debug_excerpt_docxcompose`` ermöglicht es die
-  ``docx`` Dateien, die zum Protokollauszug zusammengesetzt werden, separat in
+- Inhaltstyp Traktandum: Aktionen ``Dateien für docxcompose herunterladen``
```
Comment (deiferni) [inline docs/public/admin-manual/meeting/debugging.rst]: Aktion~~en~~ (there is only one)

File: docs/public/admin-manual/meeting/debugging.rst [hunk 4]
```diff
@@ -1,27 +1,31 @@
...
+  (In den Aktionen des Traktandums) ermöglicht es die ``docx`` Dateien,
+  die zum Protokollauszug zusammengesetzt werden, separat in
   einem ``zip`` herunterzuladen. Dies ermöglicht es Fehler im docxcompose
```
Comment (deiferni) [inline docs/public/admin-manual/meeting/debugging.rst]: im docxcompose -> in docxcompose 🤔

**Comment by deiferni** [comment]
Oh and regarding your question, tests for the manager menus would be useful, especially since these are debug views.

File: opengever/core/profiles/default/actions.xml (hunk 1 above)
Comment (njohner) [reply opengever/core/profiles/default/actions.xml]: yes that makes more sense than defining them in the generic `actions.xml` and testing for a single type

File: docs/public/admin-manual/meeting/debugging.rst (hunk 4 above)
Comment (njohner) [reply docs/public/admin-manual/meeting/debugging.rst]: I'll trust your judgment here :-D

File: opengever/meeting/browser/meetings/agendaitem.py (hunk 2 above)
Comment (njohner) [reply opengever/meeting/browser/meetings/agendaitem.py]: Yes sure. My thinking here was to name the method as what it is supposed to test, i.e. that the user is allowed to see the debug views. It just so happens that this coincides with being a manager 😃

**Pull request merged**